dpp/src/data_contract/get_property_definition_by_path.rs

```diff
@@ -0,0 +1,188 @@
+use crate::util::{json_schema::JsonSchemaExt, json_value::JsonValueExt};
+use anyhow::anyhow;
+use lazy_static::lazy_static;
+use serde_json::Value as JsonValue;
+use std::include_str;
+
+use crate::errors::ProtocolError;
+
+lazy_static! {
+    static ref BASE_DOCUMENT_SCHEMA: JsonValue =
+        serde_json::from_str(include_str!("../../schema/document/documentBase.json")).unwrap();
```

Comment:
Oh, neat! Didn't know that this could work with WASM as well, I definitely need to transition other schemas I've made before to work like that

Reply:
I was surprised as well actually. As in theory 'reads some files'.